.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: 'Build'
+
+on:
+  push:
+    paths:
+      - 'plugin.yaml'
+
+jobs:
+  release-content:
+    name: 'Create GitHub Release'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Get new version
+        id: next-version
+        run: echo "next_version=$(git diff HEAD^ HEAD --diff-filter=ACMRT -- plugin.yaml -- -U0 | awk '/^\+\s*version:/ {print $2}' | grep -Po '\d+\.\d+\.\d+')" >> $GITHUB_OUTPUT
+      - name: Display next-version output
+        run: echo "${{ steps.next-version.outputs.next_version }} ${{ steps.next-version.outputs.next_version != '' }}"
+      - name: Create tag v${{steps.next-version.outputs.next_version}}
+        if: ${{ steps.next-version.outputs.next_version != '' }}
+        uses: rickstaa/action-create-tag@v1
```

Comment:
I took a look at this action. 
ok, let's use it.
but we have to set a specific version: v1.5.4, right?